libs/solid/skeleton/src/lib/components/base-layout/base-layout.component.html

```diff
@@ -47,14 +52,13 @@
     </mat-sidenav-content>
     <mat-sidenav
       #glossary
-      [fixedInViewport]="!FixedLayout"
-      [mode]="FixedLayout ? 'side' : 'over'"
+      [fixedInViewport]="true"
```

Comment:
Brauchen wir hier explizite Variablen? Wenn sie immer den gleichen Wert haben, sind es ja Konstanten und dann kann das Verhalten auch hart codiert sein.

ABER: Ich schlage vor, dass wir die Funktionen noch drin lassen. Vielleicht gibt es ja doch neue s.o.l.i.d.-Kunden, die das Glossar immer haben wollen. Dann können wir dafür ein Flag in solid-skeleton setzen und die alte Funktion reaktivieren.

Reply:
(also: einfach so lassen, ist gut)